texts/plain/CAL-1.0

```diff
@@ -0,0 +1,137 @@
+The Cryptographic Autonomy License, v. 1.0
+
+This Cryptographic Autonomy License (the �License�) applies to any Work whose owner has marked it with any of the following notices, or a similar demonstration of intent:
+
+SPDX-License-Identifier: CAL-1.0
+Licensed under the Cryptographic Autonomy License version 1.0
+
+or
+
+SPDX-License-Identifier: CAL-1.0-Combined-Work-Exception
+Licensed under the Cryptographic Autonomy License version 1.0, with Combined Work Exceptio
```

Comment:
this looks like a typo. It's also present on https://opensource.org/licenses/CAL-1.0 but not on https://spdx.org/licenses/CAL-1.0.html

Reply:
Resolved in the latest PR